modules/adformBidAdapter.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+import {registerBidder} from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'adform';
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [],
+  isBidRequestValid: function (bid) {
+    return !!(bid.params.mid);
+  },
+  buildRequests: function (validBidRequests) {
+    var i, l, j, k, bid, _key, _value, reqParams;
+    var request = [];
+    var globalParams = [ [ 'adxDomain', 'adx.adform.net' ], [ 'fd', 1 ], [ 'url', null ], [ 'tid', null ] ];
+    var netRevenue = 'net';
+    var bids = JSON.parse(JSON.stringify(validBidRequests));
```

Comment:
we have a copy function in utils `deepClone` that could be used. Although this is probably fine.